app/Actions/SetConfig.php

```diff
@@ -2,18 +2,65 @@
 
 namespace App\Actions;
 
-use Facades\App\Paths;
+use App\InteractsWithLamboConfig;
+use Dotenv\Dotenv;
 use Illuminate\Support\Arr;
 use Illuminate\Support\Facades\File;
+use Illuminate\Support\Str;
 
 class SetConfig
 {
-    protected $existingConfig;
+    use LamboAction, InteractsWithLamboConfig;
+
+    const PROJECTPATH = 'PROJECTPATH';
+    const MESSAGE = 'MESSAGE';
+    const QUIET = 'QUIET';
+    const DEVELOP = 'DEVELOP';
+    const AUTH = 'AUTH';
+    const FRONTEND = 'FRONTEND';
+    const NODE = 'NODE';
+    const MIX = 'MIX';
+    const CODEEDITOR = 'CODEEDITOR';
+    const BROWSER = 'BROWSER';
+    const LINK = 'LINK';
+    const SECURE = 'SECURE';
+    const DB_NAME = 'DB_NAME';
+    const DB_USERNAME = 'DB_USERNAME';
+    const DB_PASSWORD = 'DB_PASSWORD';
+    const CREATE_DATABASE = 'CREATE_DATABASE';
+    const FULL = 'FULL';
+    const WITH_OUTPUT = 'WITH_OUTPUT';
+
+    public $keys = [
+        self::PROJECTPATH,
+        self::MESSAGE,
+        self::QUIET,
+        self::DEVELOP,
+        self::AUTH,
+        self::FRONTEND,
+        self::NODE,
+        self::MIX,
+        self::CODEEDITOR,
```

Comment:
i think this is `EDITOR` everywhere else in the codebase, yah?

Reply:
I named it that because it's the key used in ~/.lambo/config. The constant is only used as the key to look for when merging the saved config with the command line args. I thought it was more explicit given the use case.

Do you still want it renamed?

Reply:
ahhh got it! nope, keep it